templates/master/00-master/baremetal/files/NetworkManager-dns-vip-prepender.yaml

```diff
@@ -0,0 +1,41 @@
+filesystem: "root"
```

Comment:
SHouldn't all these be out of this PR and just in #795 Is this just to make it work and we'll rebase as soon as 795 is merged?